Microsoft.Azure.Cosmos.Samples/Usage/OpenTelemetry/Program.cs

```diff
@@ -116,6 +120,15 @@ public static async Task RunCrudDemo(Container container)
                 Console.WriteLine($"Read document with id: {i}");
             }
 
+            try
+            {
+                await container.ReadItemAsync<Item>($"random key", new PartitionKey($"random partition"));
```

Comment:
Can we add a console output to make it clear we are intentionally generating an exception here?

```suggestion
                Console.WriteLine("Generate exception by reading an invalid key.");
                await container.ReadItemAsync<Item>($"invalid key", new PartitionKey($"invalid partition"));
```